glue-codes/fast-farm/src/FAST_Farm_Types.f90

```diff
@@ -0,0 +1,5662 @@
+!STARTOFREGISTRYGENERATEDFILE 'FAST_Farm_Types.f90'
+!
+! WARNING This file is generated automatically by the FAST registry.
+! Do not edit.  Your changes to this file will be lost.
+!
+! FAST Registry
+!*********************************************************************************************************************************
+! FAST_Farm_Types
+!.................................................................................................................................
+! This file is part of FAST_Farm.
+!
+! Copyright (C) 2012-2016 National Renewable Energy Laboratory
+!
+! Licensed under the Apache License, Version 2.0 (the "License");
+! you may not use this file except in compliance with the License.
+! You may obtain a copy of the License at
+!
+!     http://www.apache.org/licenses/LICENSE-2.0
+!
+! Unless required by applicable law or agreed to in writing, software
+! distributed under the License is distributed on an "AS IS" BASIS,
+! WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+! See the License for the specific language governing permissions and
+! limitations under the License.
+!
+!
+! W A R N I N G : This file was automatically generated from the FAST registry.  Changes made to this file may be lost.
+!
+!*********************************************************************************************************************************
+!> This module contains the user-defined types needed in FAST_Farm. It also contains copy, destroy, pack, and
+!! unpack routines associated with each defined data type. This code is automatically generated by the FAST Registry.
+MODULE FAST_Farm_Types
+!---------------------------------------------------------------------------------------------------------------------------------
+USE ElastoDyn_Types
```

Comment:
@jjonkman same as FAST_Wrapper_Types

@jjonkman Why are all of the module types included here and is it necessary? I don't see them used. This adds memory overhead when compiling so some automated systems fail (i.e. [conda-forge](https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=305724&view=logs&jobId=656edd35-690f-5c53-9ba3-09c10d0bea97&j=656edd35-690f-5c53-9ba3-09c10d0bea97&t=e5c8ab1d-8ff9-5cae-b332-e15ae582ed2d)).